### PR TITLE
Remove from sandbox projects

### PR DESCRIPTION
**Pull request by g4s8**
#498 - remove a user from sandbox projects when reputation is over 1024

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#645](https://github.com/zerocracy/farm/issues/645) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#645](https://github.com/zerocracy/farm/issues/645) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/561809659). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: src/main/resources/com/zerocracy/stk/pm/staff/awards/remove_from_sandbox.groovy [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import com.zerocracy.pmo.Catalog
+import com.zerocracy.pmo.Projects
+
+def exec(Project project, XML xml) {
+  new Assume(project, xml).type('Award points were added')
+  ClaimIn claim = new ClaimIn(xml)
+  String login = claim.param('login')
+  Awards awards = new Awards(project, login).bootstrap()
+  Farm farm = binding.variables.farm
+
```
Comment (carlosmiranda) [inline src/main/resources/com/zerocracy/stk/pm/staff/awards/remove_from_sandbox.groovy]: @g4s8 no need for empty line here

**Comment by g4s8** [comment]
@carlosmiranda fixed formatting, please look again

**Comment by carlosmiranda** [comment]
@yegor256 good to merge

**Comment by yegor256** [comment]
@g4s8 but I implemented it already yesterday :(

**Comment by 0crat** [comment]
Order was finished, quality was "acceptable": +15 points just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), total is [+1665](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
The job [#645](https://github.com/zerocracy/farm/issues/645) is now out of scope

**Pull request closed without merging**